packages/portal-contracts/src/Portal.sol

```diff
@@ -324,22 +344,30 @@ contract Portal is Owned {
         e.destScriptHash = destScriptHash;
         e.amountSatsDue = amountSats;
         e.deadline = uint128(block.timestamp + 24 hours);
-        e.escrowTok = o.stakedTok + msg.value;
+        e.escrowTok = portionOfStake + totalValue;
         e.successRecipient = o.maker;
         e.timeoutRecipient = msg.sender;
 
-        // Order matched and filled.
+        // Order matched.
         emit OrderMatched(
             escrowID,
             orderID,
             o.amountSats,
+            int128(amountSats),
             o.priceTokPerSat,
             0,
             o.maker,
             msg.sender
         );
 
-        delete orderbook[orderID];
+        o.amountSats -= int128(amountSats);
+        o.stakedTok -= portionOfStake;
+        emit OrderCancelled(o.stakedTok);
```

Comment:
We're not cancelling the order, so we don't need an `OrderCancelled` event. `OrderMatched` above looks good.

Reply:
Good catch! This was a debug statement I put in before I found console.log :D